### ISSUE TITLE
rebirthing confirmation doesnt work

### ISSUE THREAD

**Issue by bietnubba** [NachitoSMO/LCLunarRework#1]
typing 'yes' into terminal when on the rebirth screen leads to a missing keyword.

**Comment by NachitoSMO** [comment]
strange. do you have the required amount of credits?
i dont doubt its an issue on my end though, i have to replace the way rebirth is implemented in the terminal at some point

**Comment by bietnubba** [comment]
> strange. do you have the required amount of credits?

yes, 9000+ credits were held at the time of the rebirth attempt. we were able to access the rebirth confirmation screen at that point, but there was the confirmation 'yes' required to complete it. typing in yes would just lead to a missing/invalid keyword.

**Comment by NachitoSMO** [comment]
Alright, once I'm home i'll try to look into it. Probably gonna rewrite how rebirth works tbh. Thank you

**Comment by bietnubba** [comment]
hey, sorry for the trouble, 100% sure it was on my end. we got to trying this mod with a new host and the rebirth worked fine! so im pretty sure it was due to my messy mod folder...

there was an issue post rebirth where the quota didnt grow after the first rebirth quota (so it reset to 130, we finished the 130, and it stayed at 130) but it might just be a mod conflict. we'll see if we can replicate it again in different scenarios and open another issue thread if so. sorry again! (love the mod btw)
